docs/book/quick-start.md

```diff
@@ -87,32 +87,24 @@ controller!
 
 ## Create a Controller
 
-Controllers are objects that implement `Laminas\Stdlib\DispatchableInterface`. This
-means they need to implement a `dispatch()` method that takes minimally a
-`Request` object as an argument.
-
-In practice, though, this would mean writing logic to branch based on matched
-routing within every controller. As such, we've created several base controller
-classes for you to start with:
-
-- `Laminas\Mvc\Controller\AbstractActionController` allows routes to match an
-  "action". When matched, a method named after the action will be called by the
-  controller. As an example, if you had a route that returned "foo" for the
-  "action" key, the "fooAction" method would be invoked.
-- `Laminas\Mvc\Controller\AbstractRestfulController` introspects the `Request` to
-  determine what HTTP method was used, and calls a method according to that.
-  - `GET` will call either the `getList()` method, or, if an "id" was matched
+We've created several base controller classes for you to start with:
+
+- `Laminas\Mvc\Controller\AbstractActionController` matches routes to methods within this class. For example, if you had a route with the "foo" action, the "fooAction" method will be invoked.
```

Comment:
Simplify the language around route matching. It was confusing to any first-time user. Also removed quotes around "action". Too many quotes make the text confusing, especially with other quoted values.

Reply:
`action` is a route match parameter name. `action` key in route is not something special. I agree on excessive quoting and I think quotes should be replaced with backticks:

```suggestion
- `Laminas\Mvc\Controller\AbstractActionController` matches routes to methods within this class. For example, if you had a route with the route match parameter `action` set to "foo", the `fooAction()` method will be invoked.
```

Reply:
"foo", "bar" or something similar should not be used, because they are not helpful examples. 

Suggestion:

> …route match parameter `action` set to "list", the `listAction()` method will be invoked.

Reply:
I'm not a fan of the "route match parameter" mention here. At this stage of framework knowledge that we expect from a developer, parameter matching means nothing. Dropping terminology like that without fully explaining the concept makes it confusing.